src/AsyncWebSocket.h

```diff
@@ -209,6 +209,9 @@ class AsyncWebSocketClient {
     //data packets
     void message(AsyncWebSocketMessage *message){ _queueMessage(message); }
     bool queueIsFull();
+    size_t queueLength() { //added by ewowi
+      return _messageQueue.length();
+    }
```

Comment:
Not a great idea for these reasons:

1. Traversal is O(N), not O(1) like with std::list on c++ >= 11
2. No mutex protection to block concurrent access

The internal structure should be changed IMO because it is not efficient.

In the community maintained fork (https://github.com/mathieucarbou/ESPAsyncWebServer), which is far more evolved than this fork, we have implemented it like this:

```c++
size_t AsyncWebSocketClient::queueLen() const {
#ifdef ESP32
  std::lock_guard<std::mutex> lock(_lock);
#endif

  return _messageQueue.size();
}
```

`_messageQueue` being a `std::list`